src/components/molecules/UiFormField/UiFormField.vue

```diff
@@ -216,6 +216,10 @@ const handleErrorEmit = (error: string | null) => {
 
   &__character-counter {
     margin-inline: auto 0;
+
+    &--has-error {
```

Comment:
Q: why did you add a class for UiFormFieldCharacterCounter to UiFormField? IMO it's a part of UiFormFieldCharacterCounter. You use it in it.

Reply:
It was added here in the first place, I just followed this pattern: the `__character-counter`  was added here while it's used in the child component. Am I wrong?

Reply:
Yes you are wrong 🙈; It'a a part of character counter; in parent, you should use "positioning properties" like margins, padding etc.